### DISCUSSION TITLE
How to change IP 172.30.XX.XX to 192.168.XX.XX

### DISCUSSION THREAD

**Discussion by Indirectelex** [microsoft/WSL#8756] [Q&A]
Hey there I need to know How to change IP 172.30.XX.XX to 192.168.XX.XX
![WindowsTerminal_qI0EVb4Jcg](https://user-images.githubusercontent.com/41977588/186026644-aa4925a1-018f-4b21-ba64-faf9ec6c9e41.png)

**Comment by NotTheDr01ds** [comment]
* What are the `ping`s intended to demonstrate?
* Providing a use-case on why you want or need to change the IP address might help in developing an answer that meets your needs.
